### PR TITLE
Filter out too far vehicles to rent

### PR DESCRIPTION
**Pull request by AdamWiktor**
To be completed by pull request submitter:

- [ ] **issue**: Link to or create an [issue](https://github.com/opentripplanner/OpenTripPlanner/issues) that describes the relevant feature or bug. Add [GitHub keywords](https://help.github.com/articles/closing-issues-using-keywords/) to this PR's description (e.g., `closes #45`).
- [ ] **roadmap**: Check the [roadmap](https://github.com/orgs/opentripplanner/projects/1) for this feature or bug. If it is not already on the roadmap, PLC will discuss as part of the review process.
- [ ] **tests**: Have you added relevant test coverage? Are all the tests passing on [the continuous integration service (Travis CI)](https://github.com/opentripplanner/OpenTripPlanner/blob/master/docs/Developers-Guide.md#continuous-integration)?
- [ ] **formatting**: Have you followed the [suggested code style](https://github.com/opentripplanner/OpenTripPlanner/blob/master/docs/Developers-Guide.md#code-style)? 
- [ ] **documentation**: If you are adding a new configuration option, have you added an explanation to the [configuration documentation](docs/Configuration.md) tables and sections?
- [ ] **changelog**: add a bullet point to the [changelog file](https://github.com/opentripplanner/OpenTripPlanner/blob/master/docs/Changelog.md) with description and link to the linked issue

To be completed by @opentripplanner/plc:

- [ ] reviews and approvals by 2 members, ideally from different organizations
- [ ] **after merging**: update the relevant card on the [roadmap](https://github.com/orgs/opentripplanner/projects/1)

### CODE REVIEW

File: src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java [hunk 1]
```diff
@@ -16,19 +16,28 @@
...
 public class SharedVehiclesUpdater extends PollingGraphUpdater {
-    SimpleStreetSplitter simpleStreetSplitter;
-    GraphUpdaterManager graphUpdaterManager;
-    VehiclePositionsGetter vehiclePositionsGetter;
-    Graph graph;
+
+    private static Logger LOG = LoggerFactory.getLogger(SharedVehiclesUpdater.class);
+
+    private static final double MAX_RADIUS = 500;
+    private static final double MAX_DISTANCE = 200;
```
Comment (MironSz) [inline src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java]: Jak to są dwie liczby, to jest trochę bez sensu. NAjpierw znajdujemy vertexy w promieniu 500 metrów, następnie zapuszczamy na tym filtry i reduce'y a potem robimy checka czy znaleziony node nie jest dalej niż 200 metrów.

**Review by MironSz — COMMENTED** [review]
No trzeba ten jeden drobiazg poprawić

File: src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java (hunk 1 above)
Comment (AdamWiktor) [reply src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java]: Te liczby nie są tym samym - pierwsza z nich to maksymalna dopuszczalna odległość do krawędzi, a druga to maksymalna dopuszczalna odległość do wierzchołka przy którym "zaparkujemy" pojazd. Nazwę je inaczej, ale moim zdaniem to powinny być dwie liczby. Wartości są do ustalenia jak dodamy faktyczne dane.

File: src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java [hunk 2]
```diff
@@ -16,19 +16,28 @@
...
 
 public class SharedVehiclesUpdater extends PollingGraphUpdater {
-    SimpleStreetSplitter simpleStreetSplitter;
-    GraphUpdaterManager graphUpdaterManager;
-    VehiclePositionsGetter vehiclePositionsGetter;
-    Graph graph;
+
+    private static Logger LOG = LoggerFactory.getLogger(SharedVehiclesUpdater.class);
+
+    private static final double MAX_DISTANCE_TO_EDGE = 500;
```
Comment (MironSz) [inline src/main/java/org/opentripplanner/updater/vehicle_sharing/SharedVehiclesUpdater.java:34]: gituwa

**Review by MironSz — APPROVED** [review]

**Pull request merged**